static/docs/get-started/add-files.md

```diff
@@ -26,6 +23,10 @@ example:
 $ dvc add data/data.xml
 ```
 
+> For an alternative and faster way to download the project's dataset in a
```

Comment:
I think this is misleading - import serves a different purpose, it's not just a faster (btw why is it faster?) way. Also, I would not include too many notes, especially that go forward. Let's keep it simple for now.

Reply:
OK, removed note (:

Reply:
p.s. by "faster" I meant that typing a single command consumes less of the user's time.